frontend/.env.test

```diff
@@ -0,0 +1 @@
+REACT_APP_BASE_URL=http://my.test.serv
```

Comment:
```suggestion
REACT_APP_BASE_URL=http://localhost:8080
```

Pretty sure you don't have a testserver :D

Reply:
Yes, i'm also sure :)